.gitignore

```diff
@@ -1,6 +1,6 @@
 #NPM
 node_modules
-npm-debug.log
+npm-debug.log*
```

Comment:
@jamesrichford `npm-debug` files are often saved with a timestamp on the file type (don't ask me why). e.g. `npm-debug.log12312238141`. At the moment they're not ignored from git